### DISCUSSION TITLE
Issue with images and GIFs not loading on GitHub website but working on mobile application

### DISCUSSION THREAD

**Discussion by VISHRUT225** [community/community#56614] [Repositories]
### Select Topic Area

Bug

### Body

I am experiencing an issue with my GitHub account where images and GIFs are not loading on the GitHub website. Interestingly, they load perfectly fine on the mobile application. I would like to report this issue to the GitHub support team for further investigation and resolution. Could you please assist me in addressing this matter? Thank you.
![image](https://github.com/community/community/assets/102377338/1e51f183-3cf3-4303-9d25-4b06de6f6005)

**Comment by VikasNU1307** [comment]
I'm also facing same issue😥

**Comment by Amenvast43** [comment]
How to delete repositories
